### PR TITLE
Bug 1936859: Fix oVirt release 4.4 and 4.5 workflows

### PR DESCRIPTION
**Pull request by Gal-Zaidman**
- Use ignition 2.2.0 for releases 4.4 and 4.5 see discussion on https://github.com/openshift/release/pull/15521#discussion_r570107580
- Remove monoitorying PVC from 4.4 and 4.5 since there is no CSI on those releases

This should fix:
- https://bugzilla.redhat.com/show_bug.cgi?id=1936859
- https://bugzilla.redhat.com/show_bug.cgi?id=1936857

### CODE REVIEW

**Comment by Gal-Zaidman** [comment]
The jobs are failing on tests but the cluster installation passes

**Comment by vrutkovs** [comment]
/retitle Bug 1936859: Fix oVirt release 4.4 and 4.5 workflows

**Comment by Gal-Zaidman** [comment]
/hold

I want to test this change a bit more

**Comment by Gal-Zaidman** [comment]
/unhold

We can see that a 4.5 job finished successfully.
I assume that the failures are duo to the fact that so many jobs started at the same time

File: ci-operator/step-registry/ipi/ovirt/pre/e2e/release-4.5-4.4/ipi-ovirt-pre-e2e-release-4.5-4.4-chain.yaml
Comment (vrutkovs) [inline ci-operator/step-registry/ipi/ovirt/pre/e2e/release-4.5-4.4/ipi-ovirt-pre-e2e-release-4.5-4.4-chain.yaml:2]:
```suggestion
  as: ipi-ovirt-pre-e2e-no-monitoring
```

File: ci-operator/step-registry/ipi/ovirt/pre/upgrade/release-4.5-4.4/ipi-ovirt-pre-upgrade-release-4.5-4.4-chain.yaml
Comment (vrutkovs) [inline ci-operator/step-registry/ipi/ovirt/pre/upgrade/release-4.5-4.4/ipi-ovirt-pre-upgrade-release-4.5-4.4-chain.yaml:2]:
```suggestion
  as: ipi-ovirt-pre-upgrade-no-monitoring-pvc
```

File: ci-operator/step-registry/openshift/e2e/ovirt/conformance/release-4.4-4.5/openshift-e2e-ovirt-conformance-release-4.4-4.5-workflow.yaml
Comment (vrutkovs) [inline ci-operator/step-registry/openshift/e2e/ovirt/conformance/release-4.4-4.5/openshift-e2e-ovirt-conformance-release-4.4-4.5-workflow.yaml:2]: Why is it renamed? `no-csi` explains the purpose of this workflow, as it may be used in other releases too

Comment (Gal-Zaidman) [reply ci-operator/step-registry/openshift/e2e/ovirt/conformance/release-4.4-4.5/openshift-e2e-ovirt-conformance-release-4.4-4.5-workflow.yaml:2]: That is what I thought in the beginning, but now we will need a specific workflow for 4.6/4.7 due to added features to the installer and that will just get more and more confusing...
We agreed that locking the workflow to the release and explaining in the description is more helpful for us

Comment (vrutkovs) [reply ci-operator/step-registry/openshift/e2e/ovirt/conformance/release-4.4-4.5/openshift-e2e-ovirt-conformance-release-4.4-4.5-workflow.yaml:2]: >we will need a specific workflow for 4.6/4.7

how many workflows do we need and why these can't be split into csi/no csi?

Comment (Gal-Zaidman) [reply ci-operator/step-registry/openshift/e2e/ovirt/conformance/release-4.4-4.5/openshift-e2e-ovirt-conformance-release-4.4-4.5-workflow.yaml:2]: We will need workflows for:
4.5 and 4.4 - no CSI
4.6 and 4.7 - no affinity groups
4.8 and above.

Its not critical but we just think it is simpler to pin it to the release name since it will not be backported

**Comment by Gal-Zaidman** [comment]
@vrutkovs
I removed the rename and I will move it into a separate PR, since it needs many approvals and I will need another renaming one for the affinity groups workflow (4.8).
Can you add lgtm?

File: ci-operator/step-registry/ipi/ovirt/pre/e2e/ipi-ovirt-pre-e2e-chain.yaml
Comment (vrutkovs) [inline ci-operator/step-registry/ipi/ovirt/pre/e2e/ipi-ovirt-pre-e2e-chain.yaml:2]:
```suggestion
  as: ipi-ovirt-pre-monitoringpvc
```
or `-withcsi` or no postfix at all. This chain doesn't run e2e in any case

File: ci-operator/step-registry/ipi/conf/ovirt/ipi-conf-ovirt-chain.yaml
Comment (vrutkovs) [inline ci-operator/step-registry/ipi/conf/ovirt/ipi-conf-ovirt-chain.yaml:7]: I'd prefer default chain to have PVC (as we release new versions there'd be only two version without CSI support, rapidly moving to EOL)

File: ci-operator/step-registry/ipi/ovirt/pre/upgrade/ipi-ovirt-pre-upgrade-chain.yaml
Comment (vrutkovs) [inline ci-operator/step-registry/ipi/ovirt/pre/upgrade/ipi-ovirt-pre-upgrade-chain.yaml:8]:
```suggestion
    The IPI setup step contains all steps that provision an OpenShift cluster with persistent Prometheus storage.
```

**Review by vrutkovs — CHANGES_REQUESTED** [review]

File: ci-operator/step-registry/ipi/ovirt/pre/e2e/ipi-ovirt-pre-e2e-chain.yaml
Comment (Gal-Zaidman) [reply ci-operator/step-registry/ipi/ovirt/pre/e2e/ipi-ovirt-pre-e2e-chain.yaml:2]: Done, I removed the prefix

File: ci-operator/step-registry/ipi/ovirt/pre/upgrade/ipi-ovirt-pre-upgrade-chain.yaml
Comment (Gal-Zaidman) [reply ci-operator/step-registry/ipi/ovirt/pre/upgrade/ipi-ovirt-pre-upgrade-chain.yaml:8]: Done

File: ci-operator/step-registry/ipi/conf/ovirt/ipi-conf-ovirt-chain.yaml
Comment (Gal-Zaidman) [reply ci-operator/step-registry/ipi/conf/ovirt/ipi-conf-ovirt-chain.yaml:7]: I prefer not to add it on the ovirt conf chain because I don't want to have multiple ovirt conf chains and multiple pre chains.
I just keep the default ovirt conf chain without a reference to the monitotingpvc (same as the etcd ramfs) and have it contain only the ovirt configuration steps, and reference the monitotingpvc and etcd ramfs steps on the pre chains

**Review by vrutkovs — APPROVED** [review]
/lgtm
/hold 

for rehearsals

File: ci-operator/step-registry/openshift/e2e/ovirt/minimal/without-csi/openshift-e2e-ovirt-minimal-without-csi-workflow.yaml
Comment (vrutkovs) [inline ci-operator/step-registry/openshift/e2e/ovirt/minimal/without-csi/openshift-e2e-ovirt-minimal-without-csi-workflow.yaml:5]:
```suggestion
    - chain: ipi-ovirt-pre-without-csi
```

Comment (Gal-Zaidman) [reply ci-operator/step-registry/openshift/e2e/ovirt/minimal/without-csi/openshift-e2e-ovirt-minimal-without-csi-workflow.yaml]: Dammit, fixed :)

**Pull request merged**
